backend/models/users.py

```
@@ -0,0 +1,10 @@
+from sqlalchemy import Column, Integer, String
+from ..database import Base
```

Pylint (3.3.7)

[error] Attempted relative import beyond top-level package

(E0402)